src/index.js

```diff
@@ -41,7 +42,7 @@ function _gaCommand(trackerNames, ...args) {
       return;
     }
 
-    internalGa(...args);
+    if (_alwaysSendToDefaultTracker) internalGa(...args);
```

Comment:
From your description of the parameter in the Readme this should be `if (_alwaysSendToDefaultTracker || !Array.isArray(trackerNames)) internalGa(...args);`.
I think @rvaidya is correct that this option should only take effect when trackerNames are not defined (like you put in the readme too).
I think with that change we should be good to go.

BTW I added you as a collaborator anyway, hopefully 2 of us with not enough time is enough to keep it maintained.